storage/localstore/mode_get.go

```diff
@@ -143,6 +144,81 @@ func (db *DB) updateGC(item shed.Item) (err error) {
 	return db.shed.WriteBatch(batch)
 }
 
+// IsPinnedFileRaw checks if a given root hash is pinned as a Raw file or not.
+// This infrmation is stored in pinFilesIndex as part of pinning the file
+func (db *DB) IsPinnedFileRaw(addr chunk.Address) (bool, error) {
+	var item shed.Item
+	item.Address = addr
+	i, err := db.pinFilesIndex.Get(item)
+	if err != nil {
+		return false, err
+	}
+	raw := false
+	if i.IsRaw > 0 {
+		raw = true
+	}
+	return raw, nil
+}
+
+// IsFilePinned checks if a given root hash is pinned or not.
+// It check for the given root hash in the pinFilesIndex
+func (db *DB) IsFilePinned(addr chunk.Address) bool {
+	var item shed.Item
+	item.Address = addr
+	has, err := db.pinFilesIndex.Has(item)
+	if err != nil {
+		return false
+	}
+	return  has
```

Comment:
lint (two spaces here)

Reply:
i don't see any spaces now.. may be i fixed it